python/ray/actor.py

```diff
@@ -668,6 +671,12 @@ def __del__(self):
         # there are ANY handles in scope in the process that created the actor,
         # not just the first one.
         worker = ray.worker.get_global_worker()
+        if (worker.mode == ray.worker.SCRIPT_MODE
```

Comment:
The `worker.mode == ray.worker.SCRIPT_MODE` check isn't really necessary, right?

Reply:
I think it's needed. because in a worker, `self._ray_actor_driver_id.id() != worker.worker_id` is always true